slack-lint/src/test/java/slack/lint/DeprecatedAnnotationDetectorTest.kt

```diff
@@ -237,6 +237,8 @@ class DeprecatedAnnotationDetectorTest : BaseSlackLintTest() {
     """
         package slack.test;
 
+        import java.lang.Deprecated;
```

Comment:
Was this necessary?

Reply:
Seemed wrong not having it and the other test having it, but moved this around when debugging an issue. Will try reverting.